client/packages/electron/forge.config.ts

```diff
@@ -2,6 +2,7 @@ import type { ForgeConfig } from '@electron-forge/shared-types';
 import { MakerSquirrel } from '@electron-forge/maker-squirrel';
 import { MakerZIP } from '@electron-forge/maker-zip';
 import { WebpackPlugin } from '@electron-forge/plugin-webpack';
+import { MakerDMG } from '@electron-forge/maker-dmg';
```

Comment:
I thought it was pretty neat. if you build on macOS it will make the DMG for you. tested running on windows and it skips the DMG creation, without causing an error.